templates/issue.sample

```diff
@@ -4,7 +4,13 @@
   <severity>Information</severity>
   <certainty>100</certainty>
   ​<description><![CDATA[<p>Netsparker detected a missing <code>X-XSS-Protection</code> header which means that this website could be at risk of a Cross-site Scripting (XSS) attacks.</p>]]></description>
+  <impact><![CDATA[ <div>;This issue is reported as additional information only. There is no direct impact arising from this issue.</div> ]]></impact>
```

Comment:
the sample is missing `<extrainformation>`, no?

Reply:
Thank you!